assets/js/modules/tagmanager/components/setup/SetupForm.js

```diff
@@ -128,6 +132,24 @@ export default function SetupForm( { finishSetup } ) {
 	// Click handler for secondary option when setting up with option to include Analytics.
 	const onSetupWithoutAnalytics = useCallback( () => submitForm(), [ submitForm ] );
 
+	let containerNames = null;
+	if ( containerID === CONTAINER_CREATE || ampContainerID === CONTAINER_CREATE ) {
+		const webContainerName = containerID === CONTAINER_CREATE
+			? <WebContainerNameTextField />
+			: null;
+
+		const ampContainerName = ampContainerID === CONTAINER_CREATE
+			? <AMPContainerNameTextField />
+			: null;
+
+		containerNames = (
+			<div className="googlesitekit-setup-module__inputs">
+				{ webContainerName }
+				{ ampContainerName }
+			</div>
+		);
+	}
```

Comment:
Similar to my above, I think we could improve readability here a bit.

Though given this code looks exactly the same as the one in the above component, seems like we could extract this logic out into its own component and use it here and in `assets/js/modules/tagmanager/components/settings/SettingsForm.js`.

Reply:
Updated to use an external component.